doc/source/rllib-algorithms.rst

```diff
@@ -23,12 +23,9 @@ Algorithm           Frameworks Discrete Actions        Continuous Actions Multi-
 `MARWIL`_           tf + torch **Yes** `+parametric`_  **Yes**            **Yes**     `+RNN`_
 `PG`_               tf + torch **Yes** `+parametric`_  **Yes**            **Yes**     `+RNN`_, `+autoreg`_
 `PPO`_, `APPO`_     tf + torch **Yes** `+parametric`_  **Yes**            **Yes**     `+RNN`_, `+autoreg`_
-`QMIX`_             torch      **Yes** `+parametric`_  No                 **Yes**     `+RNN`_
 `SAC`_              tf + torch **Yes**                 **Yes**            **Yes**
 ------------------- ---------- ----------------------- ------------------ ----------- ---------------------
-`AlphaZero`_        torch      **Yes** `+parametric`_  No                 No
 `LinUCB`_, `LinTS`_ torch      **Yes** `+parametric`_  No                 **Yes**
-`MADDPG`_           tf         **Yes**                 Partial            **Yes**
```

Comment:
Can you revert this? We link specifically to this table and it would be weird to have it missing algorithms.

Reply:
I can, but mixing multi-agent only algorithms into the same table as the others seems super confusing to me. That's why I moved it.